analytics/manager.go

```diff
@@ -55,13 +55,13 @@ func NewManager(opts Options) (Manager, error) {
 	}
 
 	var uploader uploader
-	if opts.FluentdEndpoint != "" {
+	if opts.FluentdEndpoint != "" && GCPManagedHost != opts.BaseURL.Hostname() {
```

Comment:
We do this check in multiple places, let's add a IsGCPManaged() method to Options.

Reply:
Sounds good. But to clarify, the to-be-added IsGCPManaged() method has a slightly different meaning from the one in -cli. Here it indicates if the analytics is going to be handled with `apigee.googleapis.com` instead of which type the Apigee runtime is.

Reply:
That's fine... or if you have a better name, that's cool. I'd just like to centralize and clarify that repeated check.